core/iwasm/common/arch/invokeNative_osx_universal.s

```diff
@@ -0,0 +1,13 @@
+#if defined(__aarch64__)
```

Comment:
Should add license header like other files.

Reply:
👍